### PR TITLE
Show Rate Message Issue Fixed

### LINKED ISSUE #2919: Lego message: Android.ui.rate dialog msg 1/2
**Describe the bug**

The message that invites the user to rate the app is broken into two messages, which is called "lego" in translatewiki jargon. The name "Kiwix" is inserted automatically in the middle between two messages, but this is really not great. This causes misunderstanding and incorrect translations.

**Expected behavior**

It's much better to have one message, and simply let the translators write the app name by themselves, or to insert it using a parameter.

**Steps to reproduce the behavior:**
1. Translate the rating messages at translatewiki, and there are two of them. There should be one.

**Screenshots**

**Environment**
- Version of Kiwix Android : The current thing I see at translatewiki.
- Device : 
- OS version : 

**Logs**

### PR DESCRIPTION
**Pull request by MohitMaliFtechiz**
Fixes #2919

### CODE REVIEW

**Comment by kelson42** [comment]
@MohitMaliDeveloper Sorry, now we have conflicts here :(

File: core/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -86,8 +86,7 @@
   <string name="tab_closed">Tab closed</string>
   <string name="bookmark_added">Bookmark added</string>
   <string name="rate_dialog_title">Please Rate Us</string>
-  <string name="rate_dialog_msg_1">If you enjoy using</string>
-  <string name="rate_dialog_msg_2">, please take a moment to rate it. Thanks for your support!</string>
+  <string name="rate_dialog_msg">If you enjoy using Kiwix , please take a moment to rate it. Thanks for your support!</string>
```
Comment (gouri-panda) [inline core/src/main/res/values/strings.xml:89]: Breaking it half did a great job with custom apps as well i.e message1 + app name + message 2. Is it okay to show this message `If you enjoy using Kiwix, please` to custom apps as well?
cc @kelson42

Comment (kelson42) [reply core/src/main/res/values/strings.xml:89]: @gouri-panda yes, but where is the job/string about that? If this is in the custom repo?

Comment (gouri-panda) [reply core/src/main/res/values/strings.xml:89]: This is from core. It will work both on kiwix and custom apps simultaneously.

**Comment by MohitMaliFtechiz** [comment]
> @MohitMaliDeveloper Sorry, now we have conflicts here :(

resolved

File: core/src/main/res/values/strings.xml (hunk 1 above)
Comment (kelson42) [reply core/src/main/res/values/strings.xml:89]: @gouri-panda I don't understand. Can you please open an other ticket for that? We should merge and release.

Comment (kelson42) [reply core/src/main/res/values/strings.xml:89]: I understand now, this PR can not work like this indeed. We should have a string like `If you enjoy using %s , please take a moment to rate it. Thanks for your support!`. With `%s` (this is just an example, I don't know exactly what should be the system) been replaced at runtime dynamically by the app name. This is necessary for the custom apps which have a different name.

**Review by kelson42 — CHANGES_REQUESTED** [review]
We should have a string like "If you enjoy using %s , please take a moment to rate it. Thanks for your support!." With %s (this is just an example, I don't know exactly what should be the system) been replaced at runtime dynamically by the app name. This is necessary for the custom apps which have a different name.

**Review by gouri-panda — APPROVED** [review]

**Pull request merged**